src/types/common/interfaces/common.interfaces.ts

```diff
@@ -10,13 +10,22 @@ export interface CommonEntityFields {
   updatedAt: string
 }
 export interface UserInterface {
+  _id: string
```

Comment:
Delete this UserInteface it is redundant.
We don't use them.
Use UserResponse from user.interface file.

Reply:
> Delete this UserInteface it is redundant. We don't use them. Use UserResponse from user.interface file.

allright, removed that